schedule/yam/agama_unattended_publish_qcow_s390x.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: agama_auto
+description: >
+  Prepare url for agama.auto boot parameter, boot, perform auto-installation with agama on s390x.
```

Comment:
name of the file and description of the file doesn't make any sense. Names should not contains internals like 'publish' etc.
`unattended_s390x` should be the ending of the file as usual, 'unattended' + 'arch'.

Reply:
updated.